core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -458,6 +458,78 @@ public void dayOfWeek() {
     assertEquals(integerValue(1), eval(expression));
   }
 
+  public void testDayOfWeekWithUnderscoress(FunctionExpression dateExpression, int dayOfWeek) {
```

Comment:
`private`

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/190/commits/f0448301a63c71a2f94f34fefc9e4840961cc437